src/groovy/au/org/emii/aatams/JodaOverrides.groovy

```diff
@@ -0,0 +1,42 @@
+package au.org.emii.aatams
+
+import org.joda.time.Interval
+import org.joda.time.base.AbstractInterval
+
+class JodaOverrides {
+
+    // Would be nicer to do this with an "Extension Module", but alas, that's
+    // only available in groovy 2.0.
+    //
+    // An extension module would avoid the need to mock/unmock in every test
+    // which executes the overlapsAny code.
+    static void apply() {
+        AbstractInterval.metaClass.contains = { List instantList ->
```

Comment:
Perhaps calling this 'containsAny' might make the behaviour more clear ('contains' it could be mistaken for 'containsAll').